### DISCUSSION TITLE
Media Library Pro expired

### DISCUSSION THREAD

**Discussion by dhcmega** [spatie/laravel-medialibrary#2691] [Q&A] [answered]
Hi
Can someone please tell me what is going in this case? This while running `composer install` 

```

[Composer\Downloader\TransportException]                                                                                                                                                   
  Invalid credentials for 'https://satis.spatie.be/dist/spatie/laravel-medialibrary-pro/spatie-laravel-medialibrary-pro-9632a2cd8bd43c6f74f23283c7fc7ab87679455c-zip-30f32d.zip', aborting.
```

Does this mean that I cannot continue using the last version I got? Do I have to keep paying endlessly?

Thanks

**Comment by freekmurze** [comment, marked as answer]
Our docs mention how you can continue using Media Library Pro after your license expires: https://spatie.be/docs/laravel-medialibrary/v9/handling-uploads-with-media-library-pro/installation#what-happens-when-your-license-expires

**Comment by dhcmega** [comment]
Is there any guide on how to add it to my composer? I have created my private repo but composer cannot find it. Thank you.

**Reply by fredsal** [reply]
https://packagist.org/
Read `Publishing Packages`
